core/market/decentralized/src/db/model/proposal.rs

```diff
@@ -101,6 +101,7 @@ pub struct DbProposal {
 }
 
 /// Proposal together with Negotiation object related with it.
+#[derive(Debug)]
```

Comment:
Is it necessary?

Reply:
required by `unwrap_err` which we use in tests